### PR TITLE
Added ability to rescue on different networks (phase-2-3)

### PR DESCRIPTION
**Pull request by Argeare5**
What is done:

- added `usersMerkleTrees` from https://github.com/bgd-labs/rescue-mission-phase-2-3/blob/main/js-scripts/maps/usersMerkleTrees.json (phase-2-3) (request from github)
- added support for all tokens that can be caught (icon) and added aToken icon support
- updated UI for claiming from different networks (need check UI)
- updated packages and modules

preview link: https://feat-v2.rescue-interface.pages.dev/

Questions: @kyzia551 @eboadom 

1. Do we need to update links to `about` and `github`? if yes, what links should be? (for now, `about` = phase 1 governance forum link / `github` =  github repo with contracts phase 1 link)
2. Do I need to update the readme? (include information about 2 and 3 phases)

What I want from you:

- [x] @brotherlymite let me know when contract addresses are available
- [x] @NikitaMesh you need to check the UI, the table where `assets to claim` has changed a bit, here are two addresses (0xe7d0977990B81cf52417D7964C8EB86e2793f8e7 , 0x32622Cb41c18F2a9e63F205FF11d99620D654ebD) that you can check and see how it looks now (if one asset or multiple assets on one network than one claim button, else claim button for every asset), if something is wrong or looks weird let me know
- [x] @Tima0203 look at my code changes, if anything seems weird or could be improved let me know

### CODE REVIEW

**Comment by Tima0203** [comment]
fix "check another" button is not shown here
image

File: src/web3/components/Web3Provider.tsx [hunk 1]
```diff
@@ -1,17 +1,17 @@
...
 
 export default function Web3Provider() {
   return (
     <Web3BaseProvider
       connectorsInitProps={{
         appName: 'AAVE_Rescue',
-        chains: initialChains,
-        desiredChainId: appConfig.chainId,
+        chains: internalChains,
+        desiredChainId: mainnetChainId,
```
[the remark this answers is not in the document]
Comment (Argeare5) [reply src/web3/components/Web3Provider.tsx:12]: yes, `default ` will be better, it's not desired, 

so @Tima0203 could you please update `fe-shared` repo with this changes:
`desiredChainId` -> `defaultChainId` and this should be an optional parameter if not passed use mainnet
and `wcProjectId ` -> optional

and also can you make the changes to `fe-shared` that we talked about with Andrey?

- WalletConnectV2 (pass it only networks that support it, add wcChains filtering for this)
- Fix `startTxTracking` func in `EthereumAdapter`

**Comment by brotherlymite** [comment]
@Argeare5 Should we change the color of the text (address) to something brighter as it is not so clear to see

Screenshot 2023-08-22 at 12 24 29 PM

File: src/web3/components/Web3Provider.tsx [hunk 2]
```diff
@@ -1,17 +1,17 @@
...
         urls: chainInfoHelper.urls,
+        wcProjectId: process.env.NEXT_PUBLIC_WC_PROJECT_ID || '',
```
[the remark this answers is not in the document]
Comment (Argeare5) [reply src/web3/components/Web3Provider.tsx]: fixed

**Comment by Argeare5** [comment]
@brotherlymite fixed input field, please take a look, also I'm waiting for Avalanche contract address from you

@kyzia551 @eboadom I added about modal please take a look

also I need to add T&C pre app modal like in gov v3 UI, will do tomorrow

latest link: https://d084de66.rescue-interface.pages.dev/

Снимок экрана 2023-09-18 в 17 26 12

**Comment by Argeare5** [comment]
It looks like everything is ready for merge to main, as soon as the proposal has passed, @kyzia551 or @eboadom you can merge

**Comment by brotherlymite** [comment]
@eboadom just a nitpick, should we have the new addresses on the terms and conditions also?

Screenshot 2023-09-20 at 6 38 42 PM

**Comment by eboadom** [comment]
> @eboadom just a nitpick, should we have the new addresses on the terms and conditions also?
> 
> Screenshot 2023-09-20 at 6 38 42 PM

Yes, well noticed. Can you add @Argeare5 ?

**Comment by Argeare5** [comment]
updated T&C:
Снимок экрана 2023-09-20 в 17 11 34

**Pull request merged**